nameko_slack/rtm.py

```diff
@@ -0,0 +1,109 @@
+import re
+
+import eventlet
+from nameko.extensions import Entrypoint, ProviderCollector, SharedExtension
+from slackclient import SlackClient
+
+
+EVENT_TYPE_MESSAGE = 'message'
+
+
+class SlackRTMClient(SharedExtension, ProviderCollector):
```

Comment:
I just had a look around what is the common practice, found that [PEP8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) suggests to capitalize all the letters of an abbreviation ("...`HTTPServerError` is better than `HttpServerError`"...)